xarray/core/dataset.py

```diff
@@ -8435,8 +8522,40 @@ def argmin(self: T_Dataset, dim: Hashable | None = None, **kwargs) -> T_Dataset:
         -------
         result : Dataset
 
+        Examples
+        --------
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "math_scores": (
+        ...             ["student", "test"],
+        ...             [[90, 85, 79], [78, 80, 85], [95, 92, 98]],
+        ...         ),
+        ...         "english_scores": (
+        ...             ["student", "test"],
+        ...             [[88, 90, 92], [75, 82, 79], [39, 96, 78]],
+        ...         ),
+        ...     },
+        ...     coords={
+        ...         "student": ["Alice", "Bob", "Charlie"],
+        ...         "test": ["Test 1", "Test 2", "Test 3"],
+        ...     },
+        ... )
+
+        # Indices of the minimum values along the 'student' dimension are calculated
+        >>> argmin_indices = dataset.argmin(dim="student")
+
+        # Prints student names with minimum scores for each test
+        >>> min_score_in_math = dataset["student"].values[argmin_indices["math_scores"].values]
```

Comment:
What you have done works as I asked, but it has the disadvantage of dropping out of xarray back into numpy (by calling `.values`). You don't actually need to do that, you can use `.isel` instead:

```suggestion
        >>> min_score_in_math = dataset["student"].isel(student=argmin_indices["math_scores"])
```